src/Contributors.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyleSheet, Text, ScrollView } from 'react-native';
+
+const Contributors = () => {
+    return (
+        <ScrollView
+            contentContainerStyle={{ flexGrow: 1, justifyContent: 'center', alignItems: 'center' }}
```

Comment:
Looks great! Later on we can open a task for someone to map this to a text file, as an easy reach task